### ISSUE TITLE
Getting labels from calculator

### ISSUE THREAD

**Issue by zloysmiertniy** [KevinMusgrave/pytorch-metric-learning#248]
Following your nice example on the Triplet Loss I am using the Accuracy calculator with all metrics and K number of clusters (for those that require it)

After testing I get all the metrics but where are the computed labels to get such metrics?

I tried using the embeddings found and tried several clustering methods, including Kmeans, which is one of the clustering methods used in the calculator, right? And I got no where similar to the precision score the calculator is telling me, so I wonder if I really get the metrics reported, and how this new model will really help me in future processes.

So TL;DR how do I get the labels from the Accuracy Calculator?

**Comment by KevinMusgrave** [comment]
@zloysmiertniy Sorry for the late reply.
> Following your nice example on the Triplet Loss I am using the Accuracy calculator with all metrics and K number of clusters (for those that require it)

The ```k``` argument refers to the "k" in k-nearest neighbors. This is different from k-means clustering.

> I tried using the embeddings found and tried several clustering methods, including Kmeans, which is one of the clustering methods used in the calculator, right? And I got no where similar to the precision score the calculator is telling me, so I wonder if I really get the metrics reported, and how this new model will really help me in future processes.

Could you explain in more detail how you are computing accuracy?

> So TL;DR how do I get the labels from the Accuracy Calculator?

AccuracyCalculator doesn't return labels given an input. It just computes accuracy. You can read about the accuracy metrics here: https://kevinmusgrave.github.io/pytorch-metric-learning/accuracy_calculation/#explanations-of-the-default-accuracy-metrics
